src/config/consts.ts

```diff
@@ -8,13 +8,11 @@ export enum StaleTime {
   Shortest = SECOND,
   Short = MINUTE,
   Default = 10 * MINUTE,
-  Long = HOUR,
 }
 export enum GCTime {
-  Shortest = 30 * SECOND,
-  Short = 2 * MINUTE,
-  Default = 20 * MINUTE,
-  Long = 2 * HOUR,
+  Shortest = 2 * StaleTime.Shortest,
```

Comment:
the max age should be shorter than the stale while revalidate

max age == time when you revalidate cache
stale while revalidate === max time you can still serve stale data / data that needs to be revalidated.

Reply:
ok true but i messed up on when the stale while revlidate starts. it only begins when max age ends